### ISSUE TITLE
developmental cataracts

### ISSUE THREAD

**Issue by sbello** [mgijax/mammalian-phenotype-ontology#3499]
There are several existing annotations to cataracts that mention "congenital" in the notes so we should consider adding this term

see http://www.informatics.jax.org/vocab/hp_ontology/HP:0000519
J:82971
J:133634

**Comment by sbello** [comment]
term label - copy HPO
"present at birth" in definition
include congenital as a synonym

**Comment by anna-anagnostop** [comment]
Added new term:
developmental cataract MP:0031345
DEF: a lens opacity that is present before or at birth
SYN: congenital cataract
SYN: developmental cataracts
SYN: congenital cataracts

Made sutural cataract MP:0010261 a direct child of developmental cataract MP:0031345
